### DISCUSSION TITLE
2006/03/31/%E5%9C%A8-labview-%E4%B8%AD%E4%BD%BF%E7%94%A8%E5%B8%B8%E9%87%8F%E5%AE%9A%E4%B9%89

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#80] [Announcements]
# 2006/03/31/%E5%9C%A8-labview-%E4%B8%AD%E4%BD%BF%E7%94%A8%E5%B8%B8%E9%87%8F%E5%AE%9A%E4%B9%89

如下图所示，在C语言里，使用#define来定义一个常数是非常基本的用法。直接使用数字，时间一长，就不只到这个数字是哪来的了。而且，这种方法也便于修改在程序中多处使用的常量的值。在C++一般是用const来达到同样的目的。

https://blog.qizhen.xyz/2006/03/31/%E5%9C%A8-labview-%E4%B8%AD%E4%BD%BF%E7%94%A8%E5%B8%B8%E9%87%8F%E5%AE%9A%E4%B9%89

**Comment by ruanqizhen** [comment]
之前博客下的留言：


mcusir
2011年05月14日 00:07 编辑
我来报下料，NI公司的机密都是将别人家的技巧据为己有，常量使用WORM-Global,建议读下这个网址：
http://forums.ni.com/t5/LabVIEW/WORM-Global/m-p/1545910

回复
Qizhen
2006年04月9日 07:41 编辑
一个常量子VI中设置多个常量也不是好办法，一个VI最多设置六个输出。再多，连线就乱七八糟了，我是很重视程序框图干净整洁的。

回复
竹贤
2006年04月9日 07:34 编辑
可以在一个常量子VI中设置多个常量。

回复
yoyo
2006年04月4日 19:08 编辑
舅舅机滚的挖，乖乖！

回复
冰雪儿
2006年04月2日 17:22 编辑
这里真得有好多要学的东西啊,我会常来的!  ^ – ^ !!!

回复
竹贤
2006年04月1日 08:52 编辑
也可以把多个常量作为一个子程序，在需要常量的地方放入这个输出常量的子程序即可。要修改常量只需修改这个子程序即可。

回复
jing
2006年03月31日 14:33 编辑
我又来凑下热闹先~~~呵呵~~计算机系我既死穴~~~

回复
